samples/matter/light_bulb/README.rst

```diff
@@ -119,6 +122,69 @@ Factory data support
     :start-after: matter_door_lock_sample_factory_data_start
     :end-before: matter_door_lock_sample_factory_data_end
 
+.. _matter_light_bulb_aws_iot:
+
+AWS IoT integration
+===================
+
+The sample can be configured to communicate with `AWS IoT Core`_ to control attributes in supported clusters on the device.
+After a connection has been established, the sample will mirror these attributes in the AWS IoT shadow document.
+This makes it possible to remotely control the device using the `AWS IoT Device Shadow Service`_.
+The supported attributes are ``OnOff`` from the ``OnOff`` cluster and ``CurrentLevel`` from the ``LevelControl`` cluster.
+
+The following figure illustrates the relationship between the AWS IoT integration layer and the light bulb sample:
+
+.. figure:: /images/aws_matter_integration.svg
+   :alt: Sample implementation of the AWS IoT integration layer
+
+   AWS IoT integration layer implementation diagram
+
+The following figure illustrates the interaction with the AWS IoT shadow Service:
+
+
+.. figure:: /images/aws_matter_interaction.svg
+   :alt: Interaction with the AWS IoT shadow service
+
+   AWS IoT Shadow and Matter interaction diagram
+
+AWS IoT setup and configuration
+-------------------------------
+
+To set up an AWS IoT instance and configure the sample, complete the following steps:
+
+1. Complete the setup and configuration described in the :ref:`lib_aws_iot` documentation to get the host name, device ID, and certificates used in the connection.
+#. Set the :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` and :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_STATIC` options in the :file:`overlay-aws-iot-integration.conf` file.
+#. Import the certificates mentioned in the :file:`light_bulb/src/aws_iot_integration/certs` file.
```

Comment:
I just noticed that there is no `light_bulb/src/aws_iot_integration/certs` file, and this currently reads like such a file would be where the certificates are mentioned. Is this the target where they should be imported instead? So something like:
```suggestion
#. Import the certificates to the :file:`light_bulb/src/aws_iot_integration/certs` folder.
```

Reply:
jepp, the user needs to create this folder and add the certs. Your suggestion works. Ill add the folder and some dummy certs to make this more obvious.